src/exec/util/unit.go

```diff
@@ -27,26 +27,26 @@ const (
 	DefaultPresetPermissions os.FileMode = 0644
```

Comment:
While DefaultFilePermissions and DefaultPresetPermissions share the same value, they are different constants. I'm going to keep them separate.